src/Network/Send.pm

```diff
@@ -1511,4 +1511,42 @@ sub sendchangetitle {
 	debug "Sent Change Title.\n", "sendPacket", 2;	
 }
 
+sub sendrecallsso {
+	my ($self, $accountID) = @_;
+	$self->sendToServer($self->reconstruct({
+		switch => 'recall_sso',
+		ID => $accountID,
+	}));
+}
+
+sub sendremoveaidsso {
+	my ($self, $accountID) = @_;
+	$self->sendToServer($self->reconstruct({
+		switch => 'remove_aid_sso',
+		ID => $accountID,
+	}));
+}
+
+sub sendmarcostart {
```

Comment:
please follow the convention, use camelCase

Reply:
Fix : 46c1338